.github/workflows/package.yaml

```diff
@@ -43,8 +43,16 @@ jobs:
           python setup.py bdist_wheel
           cp dist/*.whl docker/root
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
Do you think this would affect ffmpeg performance in any way?

Reply:
It will only affect this during the build stage (hence the slow builds). It sets up a QEMU emulation to emulate arm64 on amd64 (or vice versa if Github would have arm64 runners) so it should be native performance when deployed on the corresponding architecture.